bundle-templates.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+RELEASE_VERSION=${1:-SNAPSHOT}
+eval $(sed 's/ARG/export/gp;d' Dockerfile)
+
+ARTIFACT_DIR=${PWD}
+WORKING_DIR="$(mktemp -d)"
+trap 'rm -rf -- "${WORKING_DIR}"' EXIT
+
+pushd ${WORKING_DIR}
+
+wget -i - << EOF
+https://github.com/camunda/connector-aws-lambda/releases/download/${AWS_LAMBDA_VERSION}/aws-lambda-connector.json
+https://github.com/camunda/connector-google-drive/releases/download/${GOOGLE_DRIVE_VERSION}/google-drive-connector.json
+https://github.com/camunda/connector-http-json/releases/download/${HTTP_JSON_VERSION}/http-json-connector.json
+https://github.com/camunda/connector-sendgrid/releases/download/${SENDGRID_VERSION}/sendgrid-connector.json
+https://github.com/camunda/connector-slack/releases/download/${SLACK_VERSION}/slack-connector.json
+https://github.com/camunda/connector-sqs/releases/download/${SQS_VERSION}/aws-sqs-connector.json
```

Comment:
Can you explain this magic to me, please? 😁 Where are these versions coming from? 🤔

Reply:
Are you reading the Dockerfile and extracting it from there?

Reply:
Line 6, basically?

Reply:
Yes to not keep the versions duplicated the following line extracts all lines from the dockerfile starting with `ARG` and then replace `ARG` with `export`, so that I can export them using `eval` in the current shell. The export is needed so they are in the heredoc for wget available.

```
eval $(sed 's/ARG/export/gp;d' Dockerfile)
```

Reply:
Got it, love it 👍 Thanks for the tutorial! 🙂